frontend/scss/components/organisms/help.scss

```diff
@@ -22,6 +22,8 @@
 
 .#{organism('help')} {
 
+  padding-top: 2rem;
```

Comment:
Can we not mix units here? The other properties in this component seem to use px instead of rem.

Reply:
changed